### PR TITLE
Fixed some typos and a bug in delete_root_volume().

### PR DESCRIPTION
**Pull request by kashif**
Fixed a bug in `delete_root_volume()` due to a typo, in that `delete_root_volume()` would always `sleep(5)`.

### CODE REVIEW

File: starcluster/plugins/hadoop.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def _setup_hdfs(self, node, user):
     def _setup_dumbo(self, node):
         if not node.ssh.isfile('/etc/dumbo.conf'):
             f = node.ssh.remote_file('/etc/dumbo.conf')
-            f.write('[hadoops]\nstarcluster: %s\n' % self.hadoop_home)
```
Comment (jtriley) [inline starcluster/plugins/hadoop.py:173]: this is not a typo, [hadoops] is correct in dumbo.conf (https://github.com/klbostee/dumbo/wiki/Configuration-files)

File: starcluster/plugins/xvfb.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 class XvfbSetup(ClusterSetup):
     """
-    Installs, configures, and sets up an Xvfb server
+    Installs, configures, and sets up a Xvfb server
```
Comment (jtriley) [inline starcluster/plugins/xvfb.py:7]: Not to be picky but I think 'an' is correct here given that Xvfb is pronounced with a vowel sound (ex)...

**Comment by jtriley** [comment]
Thanks a lot for taking the time to find and fix all of those typos. I made a couple notes that need addressing but overall looks good.

**Comment by kashif** [comment]
Thanks! Let me fix this and send you another request.

**Pull request closed without merging**